examples/models/llama/export_llama.py

```diff
@@ -7,11 +7,14 @@
 # Example script for exporting Llama2 to flatbuffer
 
 import logging
+import sys
 
 import torch
 
 from .export_llama_lib import build_args_parser, export_llama
 
+sys.setrecursionlimit(4096)
```

Comment:
Yes it is needed when enable use_qnn_sha. Otherwise will trigger maximum recursion depth at the prepare_pt2e funciton.

Reply:
Can you add a comment and explain the reason? Also how likely we can guard it to args.qnn only?

Reply:
Sorry for late reply. I was running an experiment about the comment.
I believe we can move this line and import sys to a qualcomm specific condition
``` patch
@ def get_quantizer_and_quant_params(args):
@@ -557,6 +557,8 @
     quantizers = get_pt2e_quantizers(pt2e_quant_params, args.so_library)
     quant_dtype = None
     if args.qnn and args.pt2e_quantize:
+        import sys
+        sys.setrecursionlimit(4096)
```
It can guard it to args.qnn only. If this one looks better, I will raise a PR to move it and add comment